sdk/tests/utilities/test_api_client_builder.py

```diff
@@ -92,9 +92,9 @@ def test_build_client_no_token_provided_config_takes_precedence(self):
                 api_configuration=api_configuration)
 
             TempFileManager.delete_temp_file(secrets_file)
+            self.assertEqual(client.configuration.access_token, "mock_access_token")
```

Comment:
Moved inside the `with` clause as the call to get the access token is only executed when you retrieve the string value from the assertion. If you leave the assert outside the `with` clause the patched variables will be missing when it executes the code to get the access/refresh token.